apps/si-id-epu50.bash

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+export PYTHONUNBUFFERED=yes
+
+# CPU usage estimate (ps aux):
+
+/usr/local/bin/sirius-ioc-si-id-epu50.py \
+    --pv-prefix SI-10SB:ID-EPU50: \
+    --drive-io-port 5050 \
+    --drive-msg-port 5052 \
+    --beaglebone-addr 10.128.110.160 \
+    --autosave-dir /ioc-logs/autosave \
+    --request-file /ioc-logs/source/autosave_epu.req \
```

Comment:
this request-file must be ephemeral or permanent?

Reply:
done